File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/PDF_Creator/Headers_and_Footers/HTML_in_Header_Footer_Description.vb
Imports System.ComponentModel
Imports System.Text

Namespace EvoHtmlToPdfDemo.PDF_Creator.Headers_and_Footers
	Partial Public Class HTML_in_Header_Footer_Description
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub HTML_in_Header_Footer_Description_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			descriptionWebBrowser.Visible = True
			Dim descriptionString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Descriptions\WinForms\PDF_Creator\Headers_and_Footers\HTML_in_Header_Footer.html")
			descriptionWebBrowser.DocumentText = descriptionString
		End Sub
	End Class
End Namespace

File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/PDF_Creator/Headers_and_Footers/HTML_in_Header_Footer_Description.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.PDF_Creator.Headers_and_Footers
{
    public partial class HTML_in_Header_Footer_Description : UserControl
    {
        public HTML_in_Header_Footer_Description()
        {
            InitializeComponent();
        }

        private void HTML_in_Header_Footer_Description_Load(object sender, EventArgs e)
        {
            descriptionWebBrowser.Visible = true;
            string descriptionString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Descriptions\WinForms\PDF_Creator\Headers_and_Footers\HTML_in_Header_Footer.html");
            descriptionWebBrowser.DocumentText = descriptionString;
        }
    }
}
